go.mod

```diff
@@ -2,35 +2,21 @@ module github.com/nautilus/gateway
 
 require (
 	github.com/99designs/gqlgen v0.11.3
-	github.com/cpuguy83/go-md2man/v2 v2.0.0 // indirect
-	github.com/gorilla/websocket v1.4.1 // indirect
 	github.com/graph-gophers/graphql-go v0.0.0-20190108123631-d5b7dc6be53b
 	github.com/graphql-go/graphql v0.7.9 // indirect
-	github.com/hashicorp/golang-lru v0.5.4 // indirect
-	github.com/inconshreveable/mousetrap v1.0.0 // indirect
 	github.com/konsorten/go-windows-terminal-sequences v1.0.2 // indirect
-	github.com/matryer/moq v0.0.0-20200125112110-7615cbe60268 // indirect
-	github.com/mitchellh/gox v1.0.1 // indirect
 	github.com/mitchellh/mapstructure v1.2.2
 	github.com/nautilus/graphql v0.0.10
 	github.com/opentracing/opentracing-go v1.1.0 // indirect
-	github.com/pkg/errors v0.9.1 // indirect
 	github.com/sirupsen/logrus v1.4.2
-	github.com/spf13/cobra v0.0.3
-	github.com/spf13/pflag v1.0.3 // indirect
+	github.com/spf13/cobra v0.0.5
 	github.com/stretchr/testify v1.4.0
-	github.com/tcnksm/ghr v0.13.0 // indirect
-	github.com/urfave/cli v1.22.2 // indirect
-	github.com/vektah/dataloaden v0.3.0 // indirect
-	github.com/vektah/gqlparser v1.1.0
 	github.com/vektah/gqlparser/v2 v2.0.1
-	golang.org/x/crypto v0.0.0-20200320181102-891825fb96df // indirect
-	golang.org/x/mod v0.2.0 // indirect
 	golang.org/x/net v0.0.0-20200320220750-118fecf932d8
 	golang.org/x/sys v0.0.0-20200321134203-328b4cd54aae // indirect
-	golang.org/x/tools v0.0.0-20200225022059-a0ec867d517c // indirect
-	golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543 // indirect
 	gopkg.in/yaml.v2 v2.2.8 // indirect
 )
 
 go 1.13
+
+replace github.com/nautilus/graphql => github.com/obukhov/graphql v0.0.11-0.20200907082515-4b587297070f
```

Comment:
I think this needs to be removed

Reply:
yeah, update the dependency to the main library instead of fork